moto/backends.py

```diff
@@ -6,7 +6,7 @@
 import moto
 
 if TYPE_CHECKING:
-    from typing_extensions import Literal
+    from typing import Literal
```

Comment:
This is OK since `Literal` is in py3.9: https://docs.python.org/3.9/library/typing.html#typing.Literal